### PR TITLE
Feature/hombre

### PR DESCRIPTION
**Pull request by andre-merzky**
implement a simple but fast BoT agent scheduler.

### CODE REVIEW

**Comment by andre-merzky** [comment]
@itomaldonado : ping

**Comment by itomaldonado** [comment]
@andre-merzky if you are ok with my review / statement I am OK with this!

**Comment by andre-merzky** [comment]
Thanks Manuel - I added some comments, and iterated over the flow once more.

File: src/radical/pilot/agent/scheduler/hombre.py [hunk 1]
```diff
@@ -23,6 +23,15 @@
 class Hombre(AgentSchedulingComponent):
     '''
     HOMBRE: HOMogeneous Bag-of-task REsource allocator.  Don't kill me...
+
+    The scheduler implementation manages a set of resources (cores).  When it
+    gets the first scheduling request, it will assume that to bre representative
```
Comment (itomaldonado) [inline src/radical/pilot/agent/scheduler/hombre.py]: Just a typo here (bre --> be, I assume).

Comment (itomaldonado) [reply src/radical/pilot/agent/scheduler/hombre.py]: (This is the *only* comment I have. everything else is good, so I will accept, assuming you will fix it~).

**Comment by andre-merzky** [comment]
Thanks for the review!  :-)

**Pull request merged**